test/run_clippy.sh

```diff
@@ -5,16 +5,17 @@ set -eou pipefail
 source ./support/ci/shared.sh
 
 toolchain="${1:-stable}"
+maybe_install_rustup
 install_rust_toolchain "$toolchain"
 
 # TODO: these should be in a shared script?
-hab pkg install core/bzip2
-hab pkg install core/libarchive
-hab pkg install core/libsodium
-hab pkg install core/openssl
-hab pkg install core/xz
-hab pkg install core/zeromq
-hab pkg install core/protobuf
+sudo hab pkg install core/bzip2
```

Comment:
I'd like to make these conditional to avoid the need to hit the network if all the packages are already locally installed (See [my comments here](https://github.com/habitat-sh/habitat/pull/6222#pullrequestreview-208220246)). It's fine to do that in a follow-up PR too.

Reply:
Ah, gotcha.  I'll add that here.